impress-org/donation-block-for-stripe
# Feature: add method for disconnecting stripe account

## Description

This adds a "disconnect" link within the Stripe settings that opens a modal. If the user enters "DISCONNECT" in the modal input then the stripe account is disconnected. This also limits the Stripe block settings to admins.

## Visuals

<img width="591" alt="image" src="https://user-images.githubusercontent.com/2024145/156485643-24b1fa58-ee62-465f-9dfe-b7ff36b00bda.png">

## Testing Instructions

1. Connect to Stripe
2. Click the "disconnect" link
3. Follow the instructions (try breaking it, too)
4. Submit form

The block settings should immediately reflect the disconnected status, allowing the user connect again.

## Pre-review Checklist
